tests/integration_test.rs

```diff
@@ -31,7 +34,22 @@ fn make_bin_path_asm(file_name: &str) -> String {
         ARTIFACTS_PATH, file_name, file_name
     )
 }
-
+struct TestDB {
+    pub ptr: Rc<RocksDB>,
+    db_path: PathBuf,
+}
+impl TestDB {
+    fn new() -> Self {
+        let db_path = PathBuf::from(format!("./.test_db.{}/", fake_rand()));
```

Comment:
Can we make sure this won't collide when running tests?

Reply:
This struct removes the db file after going out of scope: https://github.com/lambdaclass/era_vm/pull/52/files#diff-7ff815b68943f08bef79f083c09e55ca8c18db8e25033034b12fa31f1d43dab4R48

Reply:
Perfect, I was thinking about tests running in parallel, but they don't